test/unit_tests/models/test_models.py

```diff
@@ -1859,6 +1860,351 @@ def test_fc_length_eegconformer():
     assert model is not None
 
 
+# ============================================================================
+# SimpleConv Tests
+# ============================================================================
+
+@pytest.fixture
+def dilated_conv_decoder_params():
```

Comment:
Do you want to rename these to `brain_module_params` so it matches with the class name?